plugin/ochttp/stats_transport.go

```diff
@@ -126,3 +126,75 @@ func (t *tracker) Close() error {
 	t.end()
 	return t.body.Close()
 }
+
+func (h *Handler) wrapForStats(w http.ResponseWriter, r *http.Request) (http.ResponseWriter, *http.Request, ender) {
```

Comment:
Ditto here about the name `wrapForStats`

Reply:
Done.